src/components/edit/exchange-update/exchange-form-contents.js

```diff
@@ -0,0 +1,328 @@
+            <Button
+                className={styles.editWarningButton}
+                small
+                onClick={() => setEditTargetSetupDisabled(false)}
+            >
+                {targetSetup
+                    ? i18n.t('Edit target setup')
+                    : i18n.t('Edit advanced options')}
+            </Button>
+        </Warning>
+    )
+
+EnableExternalEditWarning.propTypes = {
+    editTargetSetupDisabled: PropTypes.bool,
+    setEditTargetSetupDisabled: PropTypes.func,
+    targetSetup: PropTypes.bool,
+}
+
+const RadioDecorator = ({ label, helperText, currentSelected, children }) => (
+    <Box
+        className={classnames(styles.radioBox, {
+            [styles.radioBoxSelected]: currentSelected,
+        })}
+    >
+        <div>{children}</div>
+        <div className={styles.radioBoxText}>
+            <span className={styles.radioDecoratorLabel}>{label}</span>
+            <span className={styles.radioDecoratorHelper}>{helperText}</span>
+        </div>
+    </Box>
+)
+
+RadioDecorator.propTypes = {
+    children: PropTypes.node,
+    currentSelected: PropTypes.bool,
+    helperText: PropTypes.string,
+    label: PropTypes.string,
+}
+
+export const ExchangeFormContents = React.memo(
+    ({ requestsState, setRequestEditMode, deleteRequest }) => {
+        const { input: typeInput } = useField('type', {
+            subscription: { value: true },
+        })
+        const { value: typeValue } = typeInput
+
+        const { input: authenticationType } = useField('authentication', {
+            subscription: { value: true },
+        })
+        const { value: authenticationValue } = authenticationType
+        const [showAdvanced, setShowAdvanced] = useState(false)
+
+        const [editTargetSetupDisabled, setEditTargetSetupDisabled] = useState(
+            () => typeValue === EXCHANGE_TYPES.external
+        )
+
+        return (
+            <>
+                <Subsection text={i18n.t('Basic setup')}>
+                    <div className={styles.subsectionField1000}>
+                        <Field
+                            name="name"
+                            label={i18n.t('Exchange name')}
+                            helpText={i18n.t(
+                                'A unique name helps people find the right data exchange.'
+                            )}
+                            component={InputFieldFF}
+                            validate={hasValue}
+                        />
+                    </div>
+                    <div className={styles.subsectionField}>
+                        <FieldContainer label={i18n.t('Exchange target type')}>
+                            <div className={styles.radiosContainer}>
+                                <RadioDecorator
+                                    label={i18n.t('External')}
+                                    helperText={i18n.t(
+                                        'Send date to another DHIS2 instance or data warehouse'
```

Comment:
```suggestion
                                        'Send data to another DHIS2 instance or data warehouse'
```

Took a screenshot of the config screen for a presentation and noticed this typo!